typescript/cli/src/tests/commands/helpers.ts

```diff
@@ -41,8 +45,18 @@ export const CHAIN_3_METADATA_PATH = `${REGISTRY_PATH}/chains/${CHAIN_NAME_3}/me
 
 export const WARP_CONFIG_PATH_EXAMPLE = `${EXAMPLES_PATH}/warp-route-deployment.yaml`;
 export const WARP_CONFIG_PATH_2 = `${TEMP_PATH}/${CHAIN_NAME_2}/warp-route-deployment-anvil2.yaml`;
+export const WARP_DEPLOY_OUTPUT_PATH = `${TEMP_PATH}/warp-route-deployment.yaml`;
 export const WARP_CORE_CONFIG_PATH_2 = `${REGISTRY_PATH}/deployments/warp_routes/ETH/anvil2-config.yaml`;
 
+export function getCombinedWarpRoutePath(
+  tokenSymbol: string,
+  chains: string[],
+): string {
+  return `${REGISTRY_PATH}/deployments/warp_routes/${tokenSymbol.toUpperCase()}/${chains.join(
+    '-',
+  )}-config.yaml`;
```

Comment:
the `createWarpRouteConfigId` and `BaseRegistry.getWarpRoutesPath` functions in the registry package can help you reduce duplication with Registry code here

Reply:
I haven't reused the `BaseRegistry.getWarpRoutesPath` because it is marked as protected. Thanks for the `createWarpRouteConfigId` suggestion 🔥